composer.json

```diff
@@ -26,16 +26,18 @@
     },
     "require": {
         "php": "^7.3 || ~8.0.0",
-        "webmozart/assert": "^1.10",
         "laminas/laminas-stdlib": "^3.3",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "laminas/laminas-zendframework-bridge": "^1.0",
+        "psr/cache": "1.0.1",
+        "webmozart/assert": "^1.10"
```

Comment:
Why were these added? I don't see usage of them in the diff provided...

Reply:
In the github action composer was not able to get an installable set of packages when running with PHP 7.4. After pinning the version of psr/cache it worked. The webmozart/assert might be not needed. Check this Github action run: https://github.com/laminas/laminas-hydrator/runs/3137326621